### PR TITLE
🌱🧹 Spring cleaning

### PR DESCRIPTION
**Pull request by ruksi**
wanted to do some potential `minique` changes but didn't run so smoothly locally so updated the bare minimum

### CODE REVIEW

**Review by akx — APPROVED** [review]
✨

**Review by akx — APPROVED** [review]
✨ ✨ ✨

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,7 +25,8 @@ jobs:
           - python-version: '3.9'
           - python-version: '3.10'
           - python-version: '3.11'
-          - python-version: '3.11'
+          - python-version: '3.12'
+          - python-version: '3.12'
```
Comment (hylje) [inline .github/workflows/ci.yml:29]: do we need to do python3.12 twice?

Comment (ruksi) [reply .github/workflows/ci.yml:29]: first line is just Python `3.12`

the second line is `3.12` + `install-extra: '[sentry]'`

(this was the same as in the original `3.11` and `3.11` + `install-extra: '[sentry]'`)

Comment (hylje) [reply .github/workflows/ci.yml:29]: ah right

**Pull request merged**